### ISSUE TITLE
[BUG] Hercules emulation broken 

### ISSUE THREAD

**Issue by Fenix770** [86Box/86Box#358]
Sorry for opening this new post.

In the latest commint the Hercules emulation does not work and the Hercules Plus card has distorted graphics.

![image 1](https://user-images.githubusercontent.com/31600563/52543957-7c901700-2d8c-11e9-9f9e-aed2a386c26b.png).

Thanks for your effort!

**Comment by waltje** [comment]
I see a similar prob;em on my side.

Textmode seems to work fine, graphics mode does not.

--fred

--------------------------------------------
On Sun, 2/10/19, Fenix770 <notifications@github.com> wrote:

 Subject: [86Box/86Box] [BUG] Hercules emulation broken  (#358)
 To: "86Box/86Box" <86Box@noreply.github.com>
 Cc: "Subscribed" <subscribed@noreply.github.com>
 Date: Sunday, February 10, 2019, 9:35 PM
 
 Sorry for opening this new
 post.
 In the latest commint the Hercules emulation does not
 work and the Hercules Plus card has garbage graphics.
 .
 Thanks for your effort!
 
 —
 You are receiving this because you are subscribed to this
 thread.
 Reply to this email directly, view it on
 GitHub, or mute
 the thread.

**Comment by Fenix770** [comment]
I tested other games with CGA emulation for the Hercules card:

- Arkanoid 1 ( does not detect the CGA mode even with CGA emulation installed)
- Qix => Works with MONO.COM, but I have to force the CGA video mode and the pc speaker music of the game at the intro runs too fast.
- Volfied => I run a CGA emulator called YMENCGA.COM  and then I also have to force CGA graphics mode with /C parameter at prompt, and also has a too fast music, but graphics works right.


The reason for use Hercules CGA emulation instead of native CGA emulation:

-Nostalgia:  I used to have a Hercules card with a amber phosphor monitor. Its color palette seems to be different from CGA with mono monitor, at least in 86box, Varcem, PCem, and Dosbox. 

-On Hercules there are three "real" colors white/black/grey and an artifact color not easily distinguishable from white to represent the CGA's 4th color.

-The Hercules emulation seems to be  very close to what can be seen in a  monochrome amber monitor 
 
- The Hercules video cards were more common than CGA cards on XT class machines.

From late '80 to early 1991, the Hercules card and monitor was very popular, and half of the games had no native Hercules support, so  the use of different "drivers"  that emulate CGA card was common however not all emulators worked with the all CGA games.

Later most games launched after late 1991, removed their  support of Hercules, CGA, EGA and TANDY 1000 graphics when the prices of 386 SX machines and VGA cards dropped and the became a standard hardware in newer machines.
